app/controllers/admin/settings/billing.js

```diff
@@ -0,0 +1,20 @@
+import Controller from '@ember/controller';
+import { timezones } from 'open-event-frontend/utils/dictionary/date-time';
+
+export default Controller.extend({
+  timezones,
+  updateModel() {
+    this.set('isLoading', true);
+    let adminInvoiceModel = this.model;
+    adminInvoiceModel.save()
+      .then(() => {
```

Comment:
@shreyanshdwivedi in addition to that, I think this will be modified accordingly

Reply:
Yep